packages/containers.nix

```diff
@@ -78,6 +78,27 @@ let
       };
     };
 
+    cryptsetup = dockerTools.buildImage {
+      name = "cryptsetup";
+      tag = "v${pkgs.contrast.version}";
+      copyToRoot = with pkgs; [
+        bash
+        bashInteractive
+        coreutils
+        ncurses
+        procps
+        cryptsetup
+        e2fsprogs # mkfs.ext4
+        mount
+        util-linux # blkid
+        openssl
```

Comment:
Are all of these needed?

Reply:
Combined the bash/shell related ones to busybox. Other than that, yes. I haven't measured it, but I tried not having to include openssl here, but I didn't find another option of doing hkdf "in a shell". We _might_ convert this to a Go program but we'd have to link againt libcryptsetup which is a bit of a pain in Constellation.

Reply:
The bash program does a better job at illustrating the ideas, though.